tests/jballerina-integration-test/src/test/resources/websocket/wsservices/03_push_and_onText.bal

```diff
@@ -23,7 +23,8 @@ service onTextString on new http:WebSocketListener(9080) {
     resource function onText(http:WebSocketCaller caller, string data, boolean finalFrame) {
         var returnVal = caller->pushText(data);
         if (returnVal is error) {
```

Comment:
Can't we make this check as returnVal is WebSocketError if it would eliminate the following assignment? Can we try it? Not sure if it works though